driver-core/src/main/java/com/datastax/driver/core/Configuration.java

```diff
@@ -80,6 +85,7 @@ public Configuration(Policies policies,
         this.metricsOptions = metricsOptions;
         this.queryOptions = queryOptions;
         this.nettyOptions = nettyOptions;
+        this.codecRegistry = codecRegistry;
     }
```

Comment:
Ha, I actually avoided the builder pattern on purpose, because codec registry instances are mutable. But note that the class is final, and users are not meant to mess with it (they can play with TypeCodecs now, but that's all).